src/main/webapp/content/scss/themes/_dark-variables.scss

```diff
@@ -317,6 +317,12 @@ $stat-av-sc-legend-critical: rgba(204, 0, 0, 1);
 $stat-av-sc-legend-median: rgba(127, 127, 127, 255);
 $stat-av-sc-legend-best: rgba(40, 164, 40, 1);
 
+// Component: pdf-preview.component.scss
+$pdf-preview-pdf-container-shadow: rgba(0, 0, 0, 0.1);
```

Comment:
We prefer to use predefined colours wherever possible: https://docs.artemis.cit.tum.de/dev/guidelines/client-design#color-usage

Reply:
Can I use something like 

`lighten(black, 90%);` ?

I need black color with a little opacity

Reply:
No it doesn't work for the opacity so I need an alternative for that

Reply:
You can use `scale-color` to reduce the alpha. One example from this file:

```
$monaco-editor-build-annotation-outdated-background: scale-color($gray-500, $alpha: -80%);
```

so something like

```
$pdf-preview-pdf-container-shadow: scale-color($black, $alpha: -90%);
```

?

Reply:
This works, changed